flaml/ml.py

```diff
@@ -515,27 +532,28 @@ def evaluate_model_CV(
             fit_kwargs["sample_weight"] = weight
         valid_fold_num += 1
         total_fold_num += 1
-        total_val_loss += val_loss_i
+        val_loss_folds.append(val_loss_i)
         if log_training_metric or not isinstance(eval_metric, str):
-            if isinstance(total_metric, dict):
-                total_metric = {k: total_metric[k] + v for k, v in metric_i.items()}
+            if isinstance(metric_i, dict):
+                log_metric_folds.append(metric_i)
             elif total_metric is not None:
                 total_metric += metric_i
             else:
                 total_metric = metric_i

```

Comment:
remove?

Reply:
Do we allow metric_i to be a numeric value?

Reply:
yes.